### ISSUE TITLE
cx_Freeze 6.2 - build says plotly is deprecated

### ISSUE THREAD

**Issue by vovapyc** [marcelotduarte/cx_Freeze#696]
My dependencies are:
```
plotly==4.8.2
pandas==1.0.5
dash==1.13.4
xlrd==1.2.0
numpy<1.18.3
backtrader==1.9.75.123
matplotlib==3.2.2
certifi==2020.6.20
waitress==1.4.4
```

When I build it says:
```
ImportError:
The plotly.plotly module is deprecated,
please install the chart-studio package and use the
chart_studio.plotly module instead.
```
[That means it uses plotly v3 instead of v4](https://github.com/plotly/plotly.py/issues/1660) (from my dependencies)

**Comment by marcelotduarte** [comment]
I need more information, like OS, Python from python.org or anaconda, etc
from https://plotly.com/python/getting-started/ I see that has a optional package to install:
pip install chart-studio==1.0.0

Your program runs at python without this importerror?
Can you put your setup.py?

**Comment by vovapyc** [comment]
Sure,
My system: Windows 7 Ultimate 32bit (build 1701), Python 3.8.4.
It runs with python with no errors, and it works ok on cx_freeze v6.1.

My `setup.py`:
```
from setuptools import find_packages
from cx_Freeze import setup, Executable


options = {
    'build_exe': {
        'includes': [
            'idna', 'tkinter', 'matplotlib'
        ],
        'packages': [
            'asyncio', 'flask', 'jinja2', 'dash', 'plotly', 'waitress', 'backtrader', 'pandas', 'xlrd', 'numpy', 'matplotlib', 'certifi',
        ],
        'include_files': ['assets']
    }
}

executables = [
    Executable('main.py',
               base='console',
               targetName='main.exe')
]

setup(
    name='main',
    packages=find_packages(),
    version='0.0.3',
    executables=executables,
    options=options
)
```

**Comment by mx2048** [comment]
Have you read [Version 4 Migration Guide in Python](https://plotly.com/python/v4-migration/)?
Online features (plotly.plotly) moved to chart-studio package.
I am not positive if cx_Freeze has anything to do with the issue.

**Comment by mx2048** [comment]
I was able to reproduce the error. matplotlib is appeared to be the culprit, more specifically see this [comment](https://github.com/marcelotduarte/cx_Freeze/issues/692#issuecomment-657125847).
To fix that, add `    "excludes": ['matplotlib.tests', 'numpy.random._examples', 'plotly.plotly']` in the `'build_exe'`.

In addition, I had to fix another problem when using `'packages'` in the `'build_exe'`: `TypeError: expected str, bytes or os.PathLike object, not NoneType`. In the cx_freeze file `finder.py` after line `539` I added the following code 

```
if topLevelModule.file is None:
    continue
```

Edited: actually, this package is the culprit `'mpl_toolkits'`. Just use `'excludes': ['mpl_toolkits']`
